MetaMask/metamask-extension
# add networks without ticker trait

## Explanation
We need to get a sense for how many users currently have networks set without a ticker symbol. This is important to know before we enforce the ticker symbol being set. 

## More Information
This PR adds a new user trait that will be an array of chainIds for any network they have added without a ticker.

## Manual Testing Steps
1. set the SEGMENT_WRITE_KEY to the development write key 
2. start metamask
3. check the debugger in segment to see if the trait is populated



## Pre-Merge Checklist
